zebra-state/src/service/finalized_state/disk_format/upgrade/cache_genesis_roots.rs

```diff
@@ -0,0 +1,181 @@
+//! Updating the genesis note commitment trees to cache their roots.
```

Comment:
No, that's not actually what this change does, it only needs to update the tip sprout tree if the tip is at genesis.

Otherwise, the tip already has a cached root.

Reply:
(And whatever we do with the genesis tree in the tip column family is overwrite by the actual tip tree.)